template-only-docs/Deployment.md

```diff
@@ -0,0 +1,43 @@
+# Deployment to cloud environments
+
+## Requirements
+
+- External service access: The application must be able to make network calls to the public internet.
+- AWS Cognito: The application comes with AWS Cognito enabled by default, so AWS Cognito must be set up before deploying the application.
+- Custom domain name with HTTPS support: AWS Cognito requires HTTPS for callback urls to function.
+- Access to write to temporary directory: Rails needs to be able to write out temporary files.
+- Environment variables: You must provide the application with the environment variables listed in [local.env.example](/app-rails/local.env.example).
+- Secrets: You must provide the application with a secret `SECRET_KEY_BASE`. For more information, see the [Rails Guide on Security](https://guides.rubyonrails.org/v7.1/security.html#environmental-security).
+
+## Deploying using the Platform Infrastructure Template
+
+*Note: The following will be true once https://github.com/navapbc/template-infra/pull/650 is merged.*
```

Comment:
This feels like it belongs in a release note, and not a deployment.md file? Hopefully it won't be an issue for long.